src/Netscript/WorkerScript.ts

```diff
@@ -33,7 +33,7 @@ export class WorkerScript {
   delay: number | null = null;
 
   /** Holds the Promise reject() function while the script is "blocked" by an async op */
-  delayReject?: (reason?: ScriptDeath) => void;
+  delayReject: ((reason?: ScriptDeath) => void) | undefined = undefined;
```

Comment:
Monomorphism. <https://www.builder.io/blog/monomorphic-javascript>

For objects that are hot, we want them to only have a single shape. I came across this at random while debugging something else, and decided to fix it while I was here. (Normally I'd use `null` instead of `undefined`, but this way is guaranteed to execute the same.)

Reply:
Informative link, thanks